sdks/java/io/google-cloud-platform/src/main/java/org/apache/beam/sdk/io/gcp/bigquery/BigQuerySinkMetrics.java

```diff
@@ -166,12 +168,17 @@ public static Counter appendRowsRowStatusCounter(
    * @return Counter that tracks throttled time due to RPC retries.
    */
   public static Counter throttledTimeCounter(RpcMethod method) {
+
     LabeledMetricNameUtils.MetricNameBuilder nameBuilder =
         LabeledMetricNameUtils.MetricNameBuilder.baseNameBuilder(THROTTLED_TIME);
     nameBuilder.addLabel(RPC_METHOD, method.toString());
     MetricName metricName = nameBuilder.build(METRICS_NAMESPACE);
-
-    return new DelegatingCounter(metricName, false, true);
+    // for specific method
+    Counter fineCounter = new DelegatingCounter(metricName, false, true);
+    // for overall throttling time, used by runner for scaling decision
+    Counter coarseCounter = BigQueryServicesImpl.StorageClientImpl.THROTTLING_MSECS;
+    return new NestedCounter(
+        MetricName.named(METRICS_NAMESPACE, "nested"), fineCounter, coarseCounter);
```

Comment:
Can we keep the name how it previously was? To keep observability experience consistent